### ISSUE TITLE
makeDataURI broken in py3

### ISSUE THREAD

**Issue by patroqueeet** [xhtml2pdf/xhtml2pdf#395]
latest version has

```
return "data:" + mimetype + ";base64," + "".join(base64.encodestring(data).split())
```

but in py3

`encodestring` is deprecated:

> def encodestring(s): """Legacy alias of encodebytes().""" import warnings warnings.warn("encodestring() is a deprecated alias, use encodebytes()", DeprecationWarning, 2) return encodebytes(s)

hence with `"".join(...` and the `encodestring` returning bytes it will always break.

**Comment by FriedrichFroebel** [comment]
This seems to already have been fixed in #474.

**Comment by timobrembeck** [comment]
@FriedrichFroebel Thanks for the hint!
